examples/hello_world/finance/demo.py

```diff
@@ -0,0 +1,112 @@
+                    lowest_price = cur_env_snap[last_frame_idx:int(decision_event.item):"lowest_price"][-1]
+                    adj_closing_price = cur_env_snap[last_frame_idx:int(decision_event.item):"adj_closing_price"][-1]
+                    dividends = cur_env_snap[last_frame_idx:int(decision_event.item):"dividends"][-1]
+                    splits = cur_env_snap[last_frame_idx:int(decision_event.item):"splits"][-1]
+                    #print(holding, cost, opening_price, closing_price, highest_price, lowest_price, adj_closing_price, dividends, splits)
+                    # qurey snapshot for account information
+                    total_money = env.snapshot_list['account'][last_frame_idx:0:"total_money"][-1]
+                    remaining_money = env.snapshot_list['account'][last_frame_idx:0:"remaining_money"][-1]
+                    #print("env.tick: ", env.tick, " holding: ", holding, " cost: ", cost, "total_money:", total_money, "remaining_money", remaining_money)
+
+                    if holding > 0:  # sub_engine_name -> market
+                        # action = MarketOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick)
+
+                        # limit_order
+                        # action = LimitOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick, limit=highest_price)
+
+                        # stop order
+                        # action = StopOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick, stop=lowest_price)
+
+                        # stop limit order
+                        # action = StopLimitOrder(item=decision_event.item, amount=holding,
+                        #                 direction=OrderDirection.sell, tick=env.tick, stop=highest_price, limit=lowest_price)
+
+                        # order that has life_time, default life_time is 1, if life_time > 1, order will keep live when not triggered in life_time ticks
+                        action = LimitOrder(item=decision_event.item, amount=holding,
+                                        direction=OrderDirection.sell, tick=env.tick, limit=highest_price, life_time=10)
+                    else:
+
+                        action = MarketOrder(item=decision_event.item, amount=1000,
+                                        direction=OrderDirection.buy, tick=env.tick)
+                    print(f"    >>>>trade order: {action.id},tick:{env.tick}")
+
+                elif decision_event.action_type == ActionType.cancel_order:
+                    # cancel order decision
+                    print(f"<<<<    Cancel order decision_event:{[x.id for x in decision_event.action_scope.available_orders]},tick:{env.tick}")
+                    if len(decision_event.action_scope.available_orders) > 0:
```

Comment:
The code snippet of different order types should put to the docs, and add a ref link here.